contrib/kafka/config/README.md

```diff
@@ -24,7 +23,7 @@ Deployment steps:
      namespace: knative-eventing
    data:
      # Broker URL's for the provisioner
-     bootstrap_servers: kafkabroker.kafka:9092
+     bootstrap_servers: my-cluster-kafka-bootstrap.mynamespace:9092
```

Comment:
Mention `mynamespace` need to be replaced with namespace where kafka is installed typically `kafka`?

Reply:
I think that's implied, but sometimes we make these `REPLACEME` strings in all caps to call them out better. I don't know if that's needed here.

I would standardize between `my-thing-with-hyphens` and `mythinghyphensareoverrated`.